common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -3697,7 +3700,9 @@ public static enum ConfVars {
         "A list of io exception handler class names. This is used\n" +
         "to construct a list exception handlers to handle exceptions thrown\n" +
         "by record readers"),
-
+    HIVE_MAPRED_INPUT_DIR("mapred.input.dir", "", "Path to input dir."),
```

Comment:
move it to HiveTableUtils constants

Reply:
it's in constants now, why wasn't it removed from here?

Reply:
Fixed.